bulk/sources/bulk.move

```diff
@@ -78,4 +78,33 @@ module bulk::bulk {
         bulk_migrate_domain(user, migrate_domain_names);
         bulk_renew_domain(user, renew_domain_names, renewal_duration_secs);
     }
+
+    // ==== Registration ====
+
+    /// Subdomains only
+    public entry fun bulk_register_subdomain(
+        domain_admin: &signer,
+        domain_names: vector<String>,
+        subdomain_names: vector<String>,
+        expiration_time_secs: vector<u64>,
+        expiration_policies: vector<u8>,
+        transferrable: vector<bool>,
+        target_addrs: vector<address>,
+        to_addrs: vector<address>,
+    ) {
+        let idx = 0;
```

Comment:
do you think we should provide so many options at once? like different `to_addr` for each subdomain? if so , do you think we should add a check to make sure every vector has same length?